devauth/devauth.go

```diff
@@ -573,6 +575,19 @@ func (d *DevAuth) VerifyToken(ctx context.Context, raw string) error {
 	return nil
 }
 
+func (d *DevAuth) GetLimit(ctx context.Context, name string) (*model.Limit, error) {
+	lim, err := d.db.GetLimit(ctx, name)
+
+	switch err {
+	case nil:
+		return lim, nil
+	case store.ErrLimitNotFound:
+		return &model.Limit{Name: name, Value: 0}, nil
```

Comment:
`nil, nil`?

Reply:
well, if the limit is not there, we set it to 0, no?

and later on, to the config default (spearate task)

Reply:
fair enough